content/docs/getting_started/visual_story/setting_up.md

```diff
@@ -1,7 +1,7 @@
 ---
 $title: Setting up
 $order: 0
-$parent: /content/docs/design/visual_story.md
+$parent: /pwa/docs/getting_started/visual_story.md
```

Comment:
Something fishy here. This should be `/content/docs/getting_started/visual_story.md`.

Reply:
I fixed this - but /content fails on my local machine